kajiki/xml_template.py

```diff
@@ -452,10 +453,53 @@ def _compile_switch(self, node):
 
         yield ir.SwitchNode(node.getAttribute("test"), *body)
 
+    @annotate
+    def _compile_match(self, node):
+        """Convert py:match nodes to their IR."""
+        if version_info < (3, 10):
+            raise XMLTemplateCompileError(
+                "at least Python 3.10 is required to use py:match directive",
```

Comment:
Nit: `at least Python 3.10 is required to use py:match directive` → `At least Python 3.10 is required to use the py:match directive`

(added "the" and changed case of first letter)